protocol/contracts/libraries/LibUnripe.sol

```diff
@@ -140,14 +140,18 @@ library LibUnripe {
         emit SwitchUnderlyingToken(unripeToken, newUnderlyingToken);
     }
 
+    /**
+     * @notice Calculates the the penalized amount of Ripe Tokens corresponding to 
+     * the amount of Unripe Tokens that are Chopped according to the current Chop Rate.
+     * The new chop rate is %Recapitalized^2.
+     */
     function _getPenalizedUnderlying(
```

Comment:
Can we remove the underscore here? think `LibUnripe._getPenalizedUnderlying` doesn't look clean imo.

Reply:
Thought this was due to the internal function naming convention? Fixed here [9c882f9](https://github.com/BeanstalkFarms/Beanstalk/pull/802/commits/9c882f9f2038fdef374bf50b14cf6821e5798b29)